noether_tpp/include/noether_tpp/tool_path_modifiers/offset_modifier.h

```diff
@@ -0,0 +1,22 @@
+#pragma once
+
+#include <noether_tpp/core/tool_path_modifier.h>
+
+namespace noether
+{
+/**
+ * @brief Offsets the tool path by a fixed distance
+ */
+class OffsetModifier : public ToolPathModifier
+{
+public:
+  OffsetModifier(Eigen::Vector3d offset);
+
+  ToolPaths modify(ToolPaths tool_paths) const override;
+
+protected:
+  /** @brief In which direction to offset the tool path */
+  const Eigen::Vector3d offset_;
+};
```

Comment:
Let's make this more general and support a 6-DoF pose offset instead of just a vector

```suggestion
/**
 * @brief Offsets the tool path by a fixed pose
 */
class OffsetModifier : public ToolPathModifier
{
public:
  OffsetModifier(Eigen::Isometry3d offset);

  ToolPaths modify(ToolPaths tool_paths) const override;

protected:
  /** @brief Fixed tool path offset */
  const Eigen::Isometry3d offset_;
};
```